networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc

```diff
@@ -40,6 +40,8 @@ Before you can configure an Ingress Controller NLB on a new AWS cluster, you mus
 
 include::modules/nw-aws-nlb-new-cluster.adoc[leveloffset=+2]
 
+include::modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc[leveloffset=+2]
```

Comment:
Again, you may want to consider adding a separate module for the Updating procedure.

Reply:
Done! Thanks for the suggestion!